libs/platform/router/src/services/default-router.service.ts

```diff
@@ -56,7 +57,7 @@ export class DefaultRouterService implements RouterService {
   }
 
   navigate(route: string): void {
-    globalThis.history.pushState({}, '', route);
+    globalThis.history.pushState({ counter: this.counter + 1 }, '', route);
```

Comment:
this logic should happen somewhere inside lit-router

Reply:
`go` can be called for various reasons unrelated to `navigate`, so it's here instead of there.

Reply:
but should't we add it to state on any navigation, not only programatic one?

Reply:
We don't need to worry about this logic anymore when switching to timestamps.

Reply:
I meant the whole adding state to route logic (not only counter increment).